StPicoEvent/StPicoEvent.h

```diff
@@ -110,6 +111,8 @@ class StPicoEvent : public TObject
   UShort_t mFillId;
   Float_t  mBField;
 
+  Int_t    mTime;//To set timestamp for St_db_Maker. This is what StMuDstMaker used to GetEvtHddr()->SetGMTime(cast into unsigned int). Peifeng Liu
```

Comment:
In St_db_Maker::Make(), GetDateTime is invoked,

const TDatime &St_db_Maker::GetDateTime() const
{
  if (!fIsDBTime) return StMaker::GetDateTime();
  //(void )printf("**** fIsDBTime is set, returning its value\n");
  return fDBTime;
}

And then, StMaker::GetDateTime() uses the event header, which is written by MuDst but not (official) PicoDst.

const TDatime  &StMaker::GetDateTime() const 
{    
   const StEvtHddr *hd = GetEvtHddr();
   return hd->GetDateTime();
}

I guess I should create a separate repository and upload test case(s)?